_layouts/member.html

```diff
@@ -33,7 +33,18 @@
 {%- endcapture %}
 
 <p class="center">
-  <a href="{{ search | relative_url }}">
+  <a href="{{ search | relative_url | uri_escape }}">
+    Search for {{ page.name | default: page.title }}'s papers on the Research page
+  </a>
+</p>
+
+{% capture search -%}
+  blog/?search={{ page.name }}
+{%- endcapture %}
+
+<!--
```

Comment:
This opening comment tag doesn't have a matching close tag which messes up the bio page footer.

Probably remove this opening comment to keep the blog link, since this site has a lot of blog posts. Or add the matching closing tag.

Reply:
Thanks for this - I'll get this addressed!